### ISSUE TITLE
Documentation - Cross-Platform-Cryptography Update OAEP SHA-2 Padding Support Matrix

### ISSUE THREAD

**Issue by miketheitguy** [dotnet/corefx#33735]
Cross Platform Cryptography document mentions that OAEP SHA-2 padding is not available on Linux. But it looks like support for this was added per: https://github.com/dotnet/corefx/issues/2523

Also appears to be called out here: https://docs.microsoft.com/en-us/dotnet/core/whats-new/dotnet-core-2-1#api-changes

And using RSA.Encrypt & Decrypt on Linux specifying OAEP-SHA256 Padding appears to work without an error (using .NET Core 2.1 and PowerShell  6.1.1).

**Comment by miketheitguy** [comment]
Haven't tested OSX......

**Comment by karelz** [comment]
@bartonjs

**Comment by bartonjs** [comment]
Yeah, it works everywhere now.  netcoreapp20 is out of support, but since netcoreapp10 still is the table should probably split the OAEP-SHA2 column into a .NET Core 1.0 and .NET Core 2.1 version.

Or just make it be point-in-time.
